lib/exq.ex

```diff
@@ -24,6 +24,7 @@ defmodule Exq do
       worker(Redix, [redis_opts, connection_opts]),
       worker(Exq.Stats.Server, [opts]),
       worker(Exq.Enqueuer.Server, [opts]),
+      worker(Exq.Middlewares.Server, [opts]),
```

Comment:
Can we change `Middlewares` everywhere to `Middleware`?  I believe `Middleware` is also the plural.